recipe/meta.yaml

```
@@ -37,17 +39,58 @@ requirements:
     - fsspec >=2021.05.0
```

```suggestion
    - fsspec >=2021.05.0
    - aiohttp # to support fsspec[http]
```
Because we want fsspec[http]
https://github.com/huggingface/evaluate/blob/main/setup.py#L52C6-L52C18
https://github.com/fsspec/filesystem_spec/blob/master/pyproject.toml#L62C10-L62C17